### DISCUSSION TITLE
How to remove empty gap at the top? + Make tablet version show rooms like mobile version?

### DISCUSSION THREAD

**Discussion by Jaz1140** [dwainscheeren/dwains-lovelace-dashboard#716] [Q&A]
![PXL_20230712_030048255~2](https://github.com/dwainscheeren/dwains-lovelace-dashboard/assets/127198952/75495321-273e-483c-b6fc-aa3861643066)

I have dwains dashboard running on 2 Samsung S7 FE tablets in my home, there is a large gap at the top that doesn't seem to serve a purpose, how can I remove it, or move my tiles and menu up?

I feel like I could see a 3 more rooms without having to scroll if it all moved up.

![Screenshot_20230712-130821](https://github.com/dwainscheeren/dwains-lovelace-dashboard/assets/127198952/d3d9a45f-23fb-476b-8c4e-f35abee28246)

![Screenshot_20230712-131024](https://github.com/dwainscheeren/dwains-lovelace-dashboard/assets/127198952/8f5992c8-c715-440f-82b3-ec053b5ff65b)

Is there a way to make the tablet version like the mobile phone version where it only shows all the rooms, and then when you click a room it changes the page to the devices in that room? See android mobile phone version above.

**Comment by cnewman402** [comment]
This is a double post, see my response in your other post.
https://github.com/dwainscheeren/dwains-lovelace-dashboard/discussions/717

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
